README.md

```diff
@@ -285,6 +285,15 @@ gcloud's active project: [my-project-id]"}
 
 ```
 
+#### `-use_http_health_check`
+
+Enables HTTP health checks for the proxy, including startup, liveness, and readiness probing.
+Requires that you configure the Kubernetes container with HTTP probes ([sample](https://github.com/GoogleCloudPlatform/cloudsql-proxy/tree/main/examples/k8s-health-check/proxy_with_http_health_check.yaml)).
```

Comment:
nit: We should use a footer link like most of the other links in this README

Reply:
Just below there's an example of this:

```
See the [example here][sidecar-example] ...
```